packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -278,6 +306,24 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     return reserveGoldBalance;
   }
 
+  function getUnfrozenReserveGoldBalance() public view returns (uint256) {
+    uint256 reserveGoldBalance = getUnfrozenBalance();
+    for (uint256 i = 0; i < otherReserveAddresses.length; i++) {
+      reserveGoldBalance = reserveGoldBalance.add(otherReserveAddresses[i].balance);
+    }
```

Comment:
Nitty: This is also found in getReserveGoldBalance() and for readability it might be worth to create a separate function getOtherReserveAddressesGoldBalance() and to then use this function in getUnfrozenReserveGoldBalance() and getReserveGoldBalance().

Reply:
Good call. Updated.